stories/topics/proc-gcp-retrieve-lb-addresses.adoc

```diff
@@ -0,0 +1,15 @@
+[id="proc-gcp-retrieve-lb-addresses"]
+
+= Retrieving the load balancers addresses
```

Comment:
I think 'Retrieving the load balancer addresses' should do. Balancers sounds a little odd.